doc/yaml_configuration.md

```diff
@@ -1045,3 +1045,25 @@ a yaml configuration file.
 
 (The British English spelling (colour) is also accepted. In yaml configuration
 files, the American spelling is the alternative that has priority.)
+
+### hide-source-directories
```

Comment:
I think this should say `hide-source-paths` instead.

Reply:
Oops, sorry!